Concrete_article_data.py

```diff
@@ -0,0 +1,54 @@
+# -*-encoding: utf-8 -
+
+
+import numpy as np
+import pandas as pd
+from sklearn import gaussian_process
+import matplotlib.pyplot as plt
+
+if __name__ == '__main__':
+
+    df = pd.read_excel('C:/users/Nicolas/PycharmProjects/Concrete_Data.xls', header=0).dropna()
+    theta_0=1
+    theta_L=1e-4    # setting hyper-parameters
+    theta_U=1e-3
+    gp = gaussian_process.GaussianProcess(theta0=theta_0, thetaL=theta_L, thetaU=theta_U)
+
+    X = np.array(df.iloc[:30,[0]]) + np.random.normal(size=(30,1), loc=0, scale=5) # adding some noise because Gp don't allow                                                                                # equality in input.
```

Comment:
Better to insert comment as a oneliner before the code...
Also `the file is` -> `the file to be`